guides/Getting-Started.md

```diff
@@ -6,13 +6,17 @@ We'll start with the installation of the compiler and Spago build tool, and then
 
 #### Installing the Compiler
 
-You'll need [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node) and to be [able to install global packages](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory) to proceed.
+You'll need to install [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node).  We recommend installing [Node.js and npm via a "node version manager"](https://docs.npmjs.com/getting-started/installing-node) to avoid issues with installing packages gloablly. If you choose to install it manually, you might experience the [`EACCES` error when installing packages globally](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory).
 
-The Purescript compiler (`purs`) can be installed with npm:
+Install the Purescript compiler (`purs`) with npm:
```

Comment:
There have been a few cases where leaving it implicit that the guide actually expects you to do things (versus just letting you know that they are possible) has caused confusion, so :+1: for explicitly telling the reader to do things like this.